frontend/packages/dev-console/integration-tests/features/build-config/shipwright-build/shipwright-build-detail-page.feature

```diff
@@ -0,0 +1,79 @@
+@odc-4981
+Feature: Shipwright build details page
+              As a user, I want check my Shipwright Build and see all related BuildRuns in a second tab.
+
+        Background:
+            Given user has installed OpenShift Pipelines Operator
+              And user is at developer perspective
+              And user has created or selected namespace "aut-shipwright-build-details"
+              And user has installed Shipwright Operator
+              And user is at Add page
```

Comment:
Woundn't it make sense to install both operators first and then switch to dev perspective?

```suggestion
            Given user has installed OpenShift Pipelines Operator
              And user has installed Shipwright Operator
              And user is at developer perspective
              And user has created or selected namespace "aut-shipwright-build-details"
              And user is at Add page
```

Reply:
I've kept namespace creation before operator installation as post-installation steps for Shipwright Operator include creating Shipwright Build in the working namespace